### DISCUSSION TITLE
Remove prefix in Search Result for calculated fields

### DISCUSSION THREAD

**Discussion by sebnunes** [microsoft-search/pnp-modern-search#1778] [Q&A] [answered]
Hi all, 

When using slots with a managed properties mapped a Column of type Calculated Field, I'm getting a prefix in front of the actual value. 

The prefix depends on the Calculated field column type. 

How can I remove this prefix in the template or search result?

Thanks.

**Comment by wobba** [comment, marked as answer]
You may e.g. use handlebar helpers string methods. https://github.com/helpers/handlebars-helpers#replace in a custom template.

**Reply by sebnunes** [reply]
Great thanks! Exactly what I was looking for! Sorry for not being able to find it by myself.

**Comment by mikezimm** [comment]
@sebnunes ,
If you do figure out the syntax, please post an example here.  I am also looking at using a calculated column in some search pages but have not gotten there yet.

Thanks!

**Reply by sebnunes** [reply]
Hi, Syntax s pretty straight forward. 

See a sample here: {{ replace (slot item @root.slots.MySlot) "MyUglyString" "MyNiceString"}}
